pkg/util/validation/limits.go

```diff
@@ -95,6 +95,8 @@ type Limits struct {
 	// TODO remove this with Mimir version 2.4
 	ActiveSeriesCustomTrackersConfigOld activeseries.CustomTrackersConfig `yaml:"active_series_custom_trackers_config" json:"active_series_custom_trackers_config" doc:"hidden"`
 	ActiveSeriesCustomTrackersConfig    activeseries.CustomTrackersConfig `yaml:"active_series_custom_trackers" json:"active_series_custom_trackers" doc:"description=Additional custom trackers for active metrics. If there are active series matching a provided matcher (map value), the count will be exposed in the custom trackers metric labeled using the tracker name (map key). Zero valued counts are not exposed (and removed when they go back to zero)." category:"advanced"`
+	// Max allowed out of order samples.
```

Comment:
This comment makes it sound line "max number of samples", but this is max age of ooo samples (when received).